### PR TITLE
New package: fcitx-hangul

### PR DESCRIPTION
**Pull request by foxlet**
Add Hangul input support for fcitx (as an alternative to ibus-hangul).

### CODE REVIEW

File: srcpkgs/fcitx-hangul/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'fcitx-hangul'
+pkgname=fcitx-hangul
+version=0.3.1
+revision=1
+build_style=cmake
+short_desc="Hangul (Korean) support for fcitx"
+homepage="https://github.com/fcitx/fcitx-hangul"
+license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/fcitx-hangul/template]: please use spdx indentifier

File: srcpkgs/fcitx-hangul/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+depends="fcitx>=4.2.9 libhangul"
```
Comment (maxice8) [inline srcpkgs/fcitx-hangul/template]: makedepends=
depends=

directly below build_style=

File: srcpkgs/fcitx-hangul/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'fcitx-hangul'
+pkgname=fcitx-hangul
+version=0.3.1
+revision=1
+build_style=cmake
+depends="fcitx>=4.2.9 libhangul"
+makedepends="libhangul-devel fcitx-devel intltool pkg-config"
```
Comment (Johnnynator) [inline srcpkgs/fcitx-hangul/template]: `intltool` `pkg-config` should be in `hostmakedepends`

File: srcpkgs/fcitx-hangul/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+short_desc="Hangul (Korean) support for fcitx"
+homepage="https://github.com/fcitx/fcitx-hangul"
+license="GPL-2.0"
```
Comment (Johnnynator) [inline srcpkgs/fcitx-hangul/template]: `GPL-2.0-or-later`

File: srcpkgs/fcitx-hangul/template (hunk 3 above)
Comment (Johnnynator) [reply srcpkgs/fcitx-hangul/template]: Also the order shoudl be
```
hostmakedepnds=""
makedepends=""
depends=""
```

File: srcpkgs/fcitx-hangul/template (hunk 4 above)
Comment (foxlet) [reply srcpkgs/fcitx-hangul/template]: No version of the SPDX identifiers were initially stated, although they're both valid.

**Comment by foxlet** [comment]
The package should be ready to merge now.

**Comment by maxice8** [comment]
squash everything into one commit

**Pull request merged**